plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2MimeRulesSource.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.maven.internal.maven2;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.sonatype.nexus.mime.MimeRule;
+import org.sonatype.nexus.mime.MimeRulesSource;
+import org.sonatype.nexus.repository.maven.MavenPath;
+import org.sonatype.sisu.goodies.common.ComponentSupport;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Maven 2 specific {@link MimeRulesSource} that specifies known and format specific file MIME types.
+ *
+ * @since 3.0
+ */
+@Named(Maven2Format.NAME)
+@Singleton
+public class Maven2MimeRulesSource
+    extends ComponentSupport
+    implements MimeRulesSource
+{
+  /**
+   * Maven POMs.
+   */
+  public static final String POM_TYPE = "application/xml";
+
+  /**
+   * Maven POMs.
+   */
+  private static final MimeRule POM_RULE = new MimeRule(true, POM_TYPE);
+
+  /**
+   * SHA1 and MD5 hashes.
+   */
+  public static final String HASH_TYPE = "text/plain";
```

Comment:
Minor: prefer the constants from `ContentTypes` over repeating "text/plain" etc. here